.yarnrc.yml

```diff
@@ -0,0 +1,7 @@
+compressionLevel: 0
+
+enableGlobalCache: true
```

Comment:
Those two flags significantly increase the speed of install, bringing it [on par](https://p.datadoghq.eu/sb/d2wdprp9uki7gfks-c562c42f4dfd0ade4885690fa719c818?from_ts=1601994427359&live=true&to_ts=1602599227359) with other package managers on cold installs (we don't really want cold installs anyway on CI, but still).